docs/README.md

```diff
@@ -23,14 +23,25 @@ launch
     or "Greetings, human."
   say "What's your name?"
 
-  when "My name is $name"
+  when "(my|the) name is $name"
+    or "(you can|you should|) call me $name"
     with name = AMAZON.US_FIRST_NAME
     say "Hey! nice to meet you, $name."
     soundEffect happy.mp3
 ```
 :::
 </Feature>
 
+
+<LatestUpdates title='Latest Updates' changeLogURL='https://github.com/alexa-games/litexa/blob/master/CHANGELOG.md'>
+
+* [New syntax for defining permutations](book/state-management.html#intents-and-utterances). In `say` strings this works as an inline version of `or` statements, while in utterances this generates every permutation and adds it to your language model.<br/>`when "(I|we) (want|would like) to (see|hear) the news (please|)"`
```

Comment:
add a `/` before the "book" in the URL.

Reply:
You know, I'm not sure about that. That would denote root, which happens to work on litexa.com, but if we ever moved it deeper, the relative reference would continue to work. Thoughts?

Reply:
Oh, I didn't know that was how it worked. I want to say that we originally had a `/` before everything because it wasn't recognized as a URL otherwise, but I don't remember. In any case, what led me to comment was the consistency - at the bottom of the readme, there's already an existing /book/... link. @joseifert do you remember why we chose one way or another?

Reply:
Ya know, the consistency argument is compelling. I'm going with uniformly absolute references for now, we can always switch later.